api-reference/_hidden/GridBaseColumn/calculateSortValue.md

```diff
@@ -13,17 +13,20 @@ The data of the row to which the cell belongs.
 The value to be used in sorting.
 
 ---
-This property accepts the name of the [data source field](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/dataSource.md '{basewidgetpath}/Configuration/#dataSource') that provides values to be used in sorting...
+This property accepts the name of the [data source field](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/dataSource.md '{basewidgetpath}/Configuration/#dataSource') that provides values by which to sort this column...
 
 ---
 ##### jQuery
 
     <!--JavaScript-->$(function() {
         $("#{widgetName}Container").dx{WidgetName}({
-            columns: [{
-                dataField: "Position", // provides values for the column
-                calculateSortValue: "isOnVacation" // provides values to be used in sorting 
-            }]
+            columns: [
+                {
+                    dataField: "Position", // provides values for the column
+                    calculateSortValue: "isOnVacation" // provides values for sorting of the `Position` column
+                },
+                "isOnVacation"
```

Comment:
You don't have to add a column to use it's data field as a `calculateSortValue`.